_api-reference/msearch-template.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: default
+title: Multi-search Template 
+nav_order: 47
+redirect_from: 
+ - /opensearch/rest-api/multi-search/
+---
+
+# Multi-search Template 
+
+**Introduced 1.0**
+{: .label .label-purple }
+
+The Multi-search Template API runs multiple search template requests in a single API request.
+
+## Path and HTTP methods
```

Comment:
Add an intro sentence like that in Multi-search API. "The Multi-search Template API uses the following paths:"